Makefile

```
@@ -43,9 +48,11 @@ info:
 	touch .new-cqf-ruler
 
 connectathon:
+        ifeq ($(strip $(CI_TOOL)),) 
 	$(info connectathon checks out a specific commit SHA in case filepaths are updated)
 	git clone https://github.com/DBCG/connectathon.git
 	cd connectathon && git checkout 52084217d33a9d9fc8d79664a535edb24557635b
+        endif
```

```suggestion
connectathon:
ifeq ($(strip $(CI_TOOL)),) 
	$(info connectathon checks out a specific commit SHA in case filepaths are updated)
	git clone https://github.com/DBCG/connectathon.git
	cd connectathon && git checkout 52084217d33a9d9fc8d79664a535edb24557635b
endif
```

I believe it is convention to align conditionals in a make with the target name, see [the manual](https://www.gnu.org/software/make/manual/html_node/Conditional-Syntax.html)